Makefile

```diff
@@ -21,7 +21,7 @@ TIMESTAMPS = .timestamps
 # Find all python files that are not inside a hidden directory (directory starting with .)
 PYTHON_FILES := $(shell find ./* -type d \( -path ./build -o -path ./dist \) -prune -false -o -type f -name "*.py" -print)
 
-PYTHON_VERSION ?= 3
+PYTHON_VERSION ?= 3.7
```

Comment:
I would leave it to `3` here this library is meant to work with every python 3 version. See line 25, it uses the `python3` link which is the linux default python 3.x version (Depending on your system installation).

Reply:
I had to change that to have a "dev_requirements.txt" that is compatible with python 3.7, as else it would only be compatiable with python 3.9 and the tests would not run in ci. But now that I have updated it, I can change it back to "3".
However, I have not installed an older version than python 3.7 on my laptop. Must I install "python 3.0" to check for compatibility with even older versions, or is this not necessary? (For security reasons, I would like not to)

Reply:
No need to test older version locally when it works with 3.7